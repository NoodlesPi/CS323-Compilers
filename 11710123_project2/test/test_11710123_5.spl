struct position{
    int x;
    int y;
    int z;
};

int main(){
    int a[5][10][20];
    struct position p;
    p.x = 5;
    p.y = 6;
    p.z = 7;
    // type 5
    p.z = 'a';
    return a[p.x][p.y][p.z];
}