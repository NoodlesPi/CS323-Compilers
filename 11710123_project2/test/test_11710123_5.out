Error type 5 at Line 13: unmatching types on both sides of assignment operator (=)
